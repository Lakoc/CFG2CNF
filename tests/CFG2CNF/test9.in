S,A,B
a,b,c
S
S->abc
S->A