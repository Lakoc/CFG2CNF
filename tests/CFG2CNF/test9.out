S,A,B,a',b',c',<bc>
a,b,c
S
S->a'<bc>
<bc>->b'c'
b'->b
c'->c
a'->a